### ISSUE TITLE
Conflict with munkres 1.1.2

### ISSUE THREAD

**Issue by WCEdison** [pele-python/pele#170]
The latest Pele build will have conflict with python 2 because of Munkres 1.1.2 will no longer support python 2.0, meaning there will be syntax error if one is trying to build using python2 (affect travis and correspondin GUI)

References with other projects that have similar problem using Munkres:
https://github.com/beetbox/beets/issues/3154 

@js850 please note this will also lead to other pull request to fail as the auto-build system will consider all fix using python2 to fail by default if Munkres version is >1.1

  Traceback (most recent call last):
  File "/home/travis/miniconda/envs/pelenv/lib/python2.7/site-packages/nose/loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "/home/travis/miniconda/envs/pelenv/lib/python2.7/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/home/travis/miniconda/envs/pelenv/lib/python2.7/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/travis/miniconda/envs/pelenv/lib/python2.7/site-packages/pele/potentials/tests/test_booth.py", line 4, in <module>
    from pele.potentials.test_functions import BoothSystem
  File "/home/travis/miniconda/envs/pelenv/lib/python2.7/site-packages/pele/potentials/test_functions/__init__.py", line 2, in <module>
    from ._beale import Beale, BealeSystem
  File "/home/travis/miniconda/envs/pelenv/lib/python2.7/site-packages/pele/potentials/test_functions/_beale.py", line 5, in <module>
    from pele.systems import BaseSystem
  File "/home/travis/miniconda/envs/pelenv/lib/python2.7/site-packages/pele/systems/__init__.py", line 161, in <module>
    from .basesystem import *
  File "/home/travis/miniconda/envs/pelenv/lib/python2.7/site-packages/pele/systems/basesystem.py", line 3, in <module>
    from pele.landscape import DoubleEndedConnect
  File "/home/travis/miniconda/envs/pelenv/lib/python2.7/site-packages/pele/landscape/__init__.py", line 61, in <module>
    from .local_connect import *
  File "/home/travis/miniconda/envs/pelenv/lib/python2.7/site-packages/pele/landscape/local_connect.py", line 4, in <module>
    from pele.transition_states import findTransitionState, minima_from_ts
  File "/home/travis/miniconda/envs/pelenv/lib/python2.7/site-packages/pele/transition_states/__init__.py", line 79, in <module>
    from ._interpolate import *
  File "/home/travis/miniconda/envs/pelenv/lib/python2.7/site-packages/pele/transition_states/_interpolate.py", line 7, in <module>
    from pele.mindist import MeasureAtomicCluster
  File "/home/travis/miniconda/envs/pelenv/lib/python2.7/site-packages/pele/mindist/__init__.py", line 111, in <module>
    from .backward_compatibility import CoMToOrigin
  File "/home/travis/miniconda/envs/pelenv/lib/python2.7/site-packages/pele/mindist/backward_compatibility.py", line 2, in <module>
    from .minpermdist_stochastic import MinPermDistCluster
  File "/home/travis/miniconda/envs/pelenv/lib/python2.7/site-packages/pele/mindist/minpermdist_stochastic.py", line 4, in <module>
    from .exact_match import StandardClusterAlignment
  File "/home/travis/miniconda/envs/pelenv/lib/python2.7/site-packages/pele/mindist/exact_match.py", line 6, in <module>
    from ._minpermdist_policies import TransformAtomicCluster, MeasureAtomicCluster
  File "/home/travis/miniconda/envs/pelenv/lib/python2.7/site-packages/pele/mindist/_minpermdist_policies.py", line 3, in <module>
    from .permutational_alignment import find_best_permutation
  File "/home/travis/miniconda/envs/pelenv/lib/python2.7/site-packages/pele/mindist/permutational_alignment.py", line 23, in <module>
    import munkres
  File "/home/travis/miniconda/envs/pelenv/lib/python2.7/site-packages/munkres.py", line 79
    def pad_matrix(self, matrix: Matrix, pad_value: int=0) -> Matrix:
                               ^
SyntaxError: invalid syntax

**Comment by js850** [comment]
Hi, @WCEdison , thanks for reporting this.  It's lucky that we have three implementations of the permutational aligment algorithm because there are now issues with two of them (munkres and hungarian).  Lets hope the final implementation doesn't fail :)

I wonder how many people still use Pele with python 2.7.

**Comment by WCEdison** [comment]
@js850 thank you for the quick reply. I will just leave this on in case someone have build error. I have been using python 2.7 a bit, only reason is PELE used to be incompatible with python 3 so I had to run previous simulation using python 2.

Also when using ubuntu (tested on ubuntu x64 18 lts) and pip install and python, it is likely they use python 2 as default, and users have to call pip3 or python3 to execute other script (meaning pip install library for certain users will have problem as they will ended up installing numpy on python 2).

A simpler universal solution is to use 3rd party system to manage all the library in case of library error, e.g anachonda which I used to learng data analytics. https://www.anaconda.com/ which would be much easier as all library are now hub to prevent multiple library conflict. use chmod -x to execute the .sh file while by install this system most common libaray are installed with it.
